lib/scope.js

```diff
@@ -110,15 +114,11 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
     var in_destructuring = null;
     var in_export;
     var tw = new TreeWalker(function(node, descend){
-        var create_a_block_scope =
-            (options.screw_ie8 && node instanceof AST_Catch) ||
```

Comment:
I don't which of `--screw-ie8` or `--support-ie8` modes it would break, but the logic for one of them will be negatively impacted. There may not be sufficient test coverage for this catch scope functionality. I think it was written before mangle functionality was added to compress tests.